articles/native-platforms/ios-objc/00-introduction.md

```diff
@@ -0,0 +1,130 @@
+
+Otherwise, if you're starting an empty project of your own, or if you want to add authentication to an existing project from yours, you have to make sure you install them:
+
+#### Carthage
+
+If you are using Carthage, add the following lines to your `Cartfile`:
+
+```ruby
+github "auth0/Lock.iOS-OSX" ~> 1.28
+github "auth0/Auth0.swift" ~> 1.2
+```
+
+Then, run `carthage bootstrap`.
+
+> For more information about Carthage usage, check [their official documentation](https://github.com/Carthage/Carthage#if-youre-building-for-ios-tvos-or-watchos).
+
+#### Cocoapods
+
+If you are using [Cocoapods](https://cocoapods.org/), add these lines to your `Podfile`:
+
+```ruby
+use_frameworks!
+pod 'Lock', '~> 1.28'
+pod 'Auth0', '~> 1.2'
+```
+
+Then, run `pod install`.
+
+> For further reference on Cocoapods, check [their official documentation](http://guides.cocoapods.org/using/getting-started.html).
+
+## Your First Steps
+
+Make sure you complete these steps before going through any tutorial:
+
+#### 1. Create an application
+
+<%= include('../../_includes/_new_app') %>_
+
+#### 2. Configure your callback URLs
+
+Go to your [Client Dashboard](${manage_url}/#/applications/${account.clientId}/settings/${account.clientId}/settings) and make sure that *Allowed Callback URLs* contains the following:
+
+```shell
+{YOUR_APP_BUNDLE_IDENTIFIER}://${account.domain}/ios/{YOUR_APP_BUNDLE_IDENTIFIER}/callback
+```
+
+In your application's `Info.plist` file, register your iOS Bundle Identifier as a custom scheme. To do so, open the `Info.plist` as source code, and add this chunk of code under the main `<dict>` entry:
+
+```xml
+<key>CFBundleURLTypes</key>
+<array>
+    <dict>
+        <key>CFBundleTypeRole</key>
+        <string>None</string>
+        <key>CFBundleURLName</key>
+        <string>auth0</string>
+        <key>CFBundleURLSchemes</key>
+        <array>
+            <string>{YOUR_APP_BUNDLE_IDENTIFIER}</string>
+        </array>
+    </dict>
+</array>
+```
+
+#### 3. Add your credentials
+
+Add your credentials in `Auth0.plist`. You have to create that file if it doesn't already exist in your project:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+  <key>ClientId</key>
+  <string>${account.clientId}</string>
+  <key>Domain</key>
+  <string>${account.namespace}</string>
+</dict>
+</plist>
+```
+
+## Done!
+
+**You're all set up to start integrating Auth0 in your app!** 🎉
+
+The tutorial guides contained in this section will teach you the different things you can do with it. It's recommendable that you follow them sequentially in order, as they are progressive and there is content that usually needs to be learned from previous tutorials.
+
+Said that, we suggest you to start with the [login tutorial](01-login.md)...
```

Comment:
Should be "That said," rather than "Said that," or you could remove and start with "We suggest...".

Also in the paragraph above this, I'd probably use "It's recommended" - just sounds better to me, although there's nothing wrong with how it is.